public/css/dashboard.css

```diff
@@ -93,20 +93,42 @@ a {
     /*display:none;*/
 }
 
+.filter .flatpickr-input[readonly] {
+    background: transparent;
+    padding: 0 12px;
+    height: 32px;
+    line-height: 1.428571429;
+    margin-top: 0;
+    min-width: 5em;
+    color: #999;
+    }
+
 .filter_search {
-    border-radius: 15px;
-    padding: 10px;
-    margin-bottom: 20px;
+    padding: 20px 10px 20px 10px;
+    margin: 0 0 20px 0;
+    width: 100%;
```

Comment:
Having an `inline-block` AND `width: 100%` is the same as the default `display: block`, right?